tests/resources/aria-at-test-run.mjs

```diff
@@ -0,0 +1,1252 @@
+            },
+          ],
+          click: () =>
+            hooks.setCommandAssertion({commandIndex, assertionIndex, result: AssertionResultMap.FAIL_MISSING}),
+        }),
+      ],
+    });
+  }
+
+  /**
+   * @param {number} commandIndex
+   * @param {string} assertion
+   * @param {number} assertionIndex
+   */
+  function additionalAssertionResult(commandIndex, assertion, assertionIndex) {
+    const resultAdditionalAssertion = resultState.commands[commandIndex].additionalAssertions[assertionIndex];
+    return /** @type {InstructionDocumentResultsCommandsAssertion} */ ({
+      description: [
+        assertion,
+        {
+          required: true,
+          highlightRequired: resultAdditionalAssertion.highlightRequired,
+          description: "(required: mark support)",
+        },
+      ],
+      passChoice: assertionChoice(resultAdditionalAssertion, AdditionalAssertionResultMap.PASS, {
+        label: ["Good Support ", {offScreen: true, description: "for assertion"}],
+        click: () =>
+          hooks.setCommandAdditionalAssertion({
+            commandIndex,
+            additionalAssertionIndex: assertionIndex,
+            result: AdditionalAssertionResultMap.PASS,
+          }),
+      }),
+      failChoices: [
+        assertionChoice(resultAdditionalAssertion, AdditionalAssertionResultMap.FAIL_SUPPORT, {
+          label: ["No Support ", {offScreen: true, description: "for assertion"}],
+          click: () =>
+            hooks.setCommandAdditionalAssertion({
+              commandIndex,
+              additionalAssertionIndex: assertionIndex,
+              result: AdditionalAssertionResultMap.FAIL_SUPPORT,
+            }),
+        }),
+      ],
+    });
+  }
+}
+
+/**
+ * @typedef {typeof UserActionMap[keyof typeof UserActionMap]} UserAction
+ */
+
+export const UserActionMap = createEnumMap({
+  LOAD_PAGE: "loadPage",
+  OPEN_TEST_WINDOW: "openTestWindow",
+  CLOSE_TEST_WINDOW: "closeTestWindow",
+  VALIDATE_RESULTS: "validateResults",
+  CHANGE_TEXT: "changeText",
+  CHANGE_SELECTION: "changeSelection",
+  SHOW_RESULTS: "showResults",
+});
+
+/**
+ * @typedef {typeof UserObjectActionMap[keyof typeof UserObjectActionMap]} UserObjectAction
+ */
+
+export const UserObjectActionMap = createEnumMap({
+  FOCUS_UNDESIRABLE: "focusUndesirable",
+});
+
+/**
+ * @typedef {UserAction | UserActionFocusUnexpected} TestRunUserAction
+ */
+
+/**
+ * @typedef {EnumValues<typeof HasUnexpectedBehaviorMap>} HasUnexpectedBehavior
+ */
+
+export const HasUnexpectedBehaviorMap = createEnumMap({
+  NOT_SET: "notSet",
+  HAS_UNEXPECTED: "hasUnexpected",
+  DOES_NOT_HAVE_UNEXPECTED: "doesNotHaveUnexpected",
+});
+
+export const CommonResultMap = createEnumMap({
+  NOT_SET: "notSet",
+  PASS: "pass",
+});
+
+/**
+ * @typedef {EnumValues<typeof AdditionalAssertionResultMap>} AdditionalAssertionResult
+ */
+
+export const AdditionalAssertionResultMap = createEnumMap({
+  ...CommonResultMap,
+  FAIL_SUPPORT: "failSupport",
```

Comment:
nit: I noticed the values of these generated `Map`s all seem to differ between camelCase, natural language and capitalized. I figure it has to do with where their respective values will be used. Perhaps their appropriate usage could be described in their JSDoc tags?

Reply:
Yeah. The natural language, spaced and title cased, values and the all capitalized values are values as they already appear in the json object built and submitted from the test result from input. I'll add something like that to the doc tags.